packages/components/aside/src/aside.js

```diff
@@ -15,6 +16,7 @@ customElementDefineHelper(
 			return {
 				title: { type: String, attribute: 'data-title' },
 				visible: { type: Boolean, attribute: 'data-visible', reflect: true },
+				busy: { type: String, attribute: 'data-busy', reflect: true },
```

Comment:
why it's string? For the reason Jakob had issue with boolean properties?

Reply:
This is the same api in tradeshift-ui. I will show a message if the developer give us a string. if it is empty string, it will only show the spinner. If it is undefined, there will be nothing

Reply:
Then maybe it would be better to just use boolean property for it, what do you think?

Reply:
How could the developer pass a message if the property is boolean?

Reply:
For example: `data-busy="data loading..."`

Reply:
Shouldn't it be a separate property? It makes it a bit confusing in my opinion but if you think this is a better, I am okay with it, we can refactor later if we decided other way

Reply:
It is not good idea to have more properties. And we have the same property in Tradeshift-ui. All developers use it like this way. They will get confused when we separate it to two.

Reply:
I have a different opinion, and I think it make the api more clear what the property does. There are lots of new developers came and coming to Tradeshift and also 3rd party developers who are not used to the old api. If there's a room for improvement then the previous library should not stop us from doing it 👍

Reply:
But as I mentioned above I'm okay with it if you think it's a better way to do it